packages/core/tools/_focused.scss

```diff
@@ -26,13 +26,33 @@
 ///
 /// Provides an additional outline and border to clearly indicate when
 /// the target element has focus. Used for interactive input-based elements such
-/// as text inputs and radios.
+/// as text inputs.
 
 @mixin nhsuk-focused-input {
+  border: 2px solid $nhsuk-focus-text-color;
+  box-shadow: inset 0 0 0 2px;
+  outline: $nhsuk-focus-width solid $nhsuk-focus-color; /* 1 */
+  outline-offset: 0;
+}
+
+/// Focused radio input (form element)
+/// 
+/// Provides an additional outline and border to clearly indicate when
+/// the target element has focus. Used by radios.
+
+@mixin nhsuk-focused-radio {
+  border: $nhsuk-focus-width solid $nhsuk-focus-text-color;
+  box-shadow: 0 0 0 $nhsuk-focus-width $nhsuk-focus-color;
+}
+
+/// Focused checkbox input (form element)
+/// 
+/// Provides an additional outline and border to clearly indicate when
+/// the target element has focus. Used by checkbox.
+
+@mixin nhsuk-focused-checkbox {
   border: $nhsuk-focus-width solid $nhsuk-focus-text-color;
   box-shadow: 0 0 0 $nhsuk-focus-width $nhsuk-focus-color;
```

Comment:
Is `outline` not needed anymore? It was there for high contrast mode. 

```
outline: $nhsuk-focus-width solid transparent;
outline-offset: $nhsuk-focus-width;
```

Reply:
I believe we discussed this via slack. But from testing with various high contrast modes on Windows and Mac, I found no issues here